docs/source/en/core/i18n.md

```diff
@@ -0,0 +1,137 @@
+title: I18n Internationalization
+---
+
+For developing the multi-language application, build-in I18n support by [egg-i18n](https://github.com/eggjs/egg-i18n) plugin 
+
+## Default Language 
+
+Default `en-US`. Assume that we want to switch the default language to Simplified Chinese：
+
+```js
+// config/config.default.js
+exports.i18n = {
+  defaultLocale: 'zh-CN',
+};
+```
+
+## Switch Language
+
+Here we have some ways to switch the application's current language (Modified records will set to the cookie `locale`), can be used directly in the next request.
+
+Higher Priority: 
```

Comment:
Yes from the chinese translate is ok.
But I also think both okay, if refer use `Priority from high to low`, I can modify